setup.py

```diff
@@ -55,14 +56,20 @@
         (BASE_DIR, ['data/nssm_original.exe'])
     )],
 
-    install_requires=['indy-plenum==1.14.0-rc0',
+    install_requires=[# Changed the import path for the plenum package to point to indy-plenum fork on AdamBurdett Repo for parallel dev change back to regular version once PR is complete
+                       'indy-plenum @ git+https://github.com/burdettadam/indy-plenum.git@feature/indy-vdr-upgrade#egg=indy-plenum',
+                    #  'indy-plenum==1.14.0-rc0',
                     # importlib-metadata needs to be pinned to 3.10.1 because from v4.0.0 the package
                     # name ends in python3-importlib-metadata_0.0.0_amd64.deb
                     # see also build-scripts/ubuntu-2004/build-3rd-parties.sh
                     # https://github.com/hyperledger/indy-plenum/blob/eac38674252b539216be2c40bb13e53c5b70dad2/build-scripts/ubuntu-2004/build-3rd-parties.sh#L104-L106
+                      'indy_vdr==0.4.0.dev5', 
+                      'aries-askar==0.2.7', 
+                      'indy-credx==0.3.1',
```

Comment:
These are only used for testing, we do not want to add these dependencies to the main package.